common/network-common/src/main/java/org/apache/spark/network/client/TransportResponseHandler.java

```diff
@@ -81,6 +77,17 @@ public void removeFetchRequest(StreamChunkId streamChunkId) {
     outstandingFetches.remove(streamChunkId);
   }
 
+  public void addFetchAsStreamRequest(
+      StreamChunkId streamChunkId,
+      StreamCallback callback) {
+      updateTimeOfLastRequest();
```

Comment:
indent？

Reply:
Spark uses 4 space indentation for multi-line method parameters

Reply:
I mean `updateTimeOfLastRequest();`, it is not parameters and should be 2 space indentation ?

Reply:
ah missed this one!